src/core/ast.h

```diff
@@ -1021,14 +1021,14 @@ class AST_LangPrimitive : public AST_expr {
 public:
     enum Opcodes {
         ISINSTANCE,
-        LANDINGPAD,
+        LANDINGPAD, // grabs the info about the last raised exception
         LOCALS,
         GET_ITER,
         IMPORT_FROM,
         IMPORT_NAME,
         IMPORT_STAR,
         NONE,
-        NONZERO,
+        NONZERO, // Python's __nonzero__ magic method
```

Comment:
Yeah I guess so, and other similar cases (while loops, boolean expressions, whether the `__exit__` method caught the exception).